season_5/down_in_the_river_to_pray.ly

```diff
@@ -0,0 +1,729 @@
+  gis4 b4 fis16 e8.~ e4(
+
+  % page 8, system 2
+  gis4) fis8 e8 e8 e8 cis8( gis'8)
+  b1~
+  b1\fermata
+}
+
+sopranoTwoMusic = \relative c''{
+  \key c \major
+  \numericTimeSignature
+  \time 2/2
+  \tempo 2=55
+  \set Score.markFormatter = #format-mark-circle-numbers
+
+  % lead-in
+  r1
+  r1
+
+  % alto/tenor duet
+  r1 r1 r1 r1 r1 r1 r1 r1 r1
+  r1 r1 r1 r1 r1 r1 r1 r1 r1
+
+  % page 4, system 1
+  r4 b4 b4 cis4
+  e4 e8 e8 e8 e8 e4
+  e1
+  fis8 fis8 fis8 gis8 fis4 e4
+
+  % page 4, system 2
+  gis4 fis4 e4 cis4
+  b4 gis4 b4~ b8 cis8
+  e4 cis4 e4( gis4)
+  fis2 e8 e8~ e8 cis8
+
+  % page 5, system 1
+  b1
+  b2 b4 b4
+  b4 b4 a2
+  gis4 a4 b4 cis8 cis8
+
+  % page 5, system 2
+  b8 b8 b4 a2
+  \slashedGrace e16( fis2) fis4 e4
+  gis4 b4 fis16( e8.~ e4)
+  e4 fis8 gis8 a8 gis8 fis8( e8)
+
+  % page 6, system 1
+  e2. r4
+  \time 4/4 \tempo 4=90
+  r1
+  r4 b4 b4 c4
+  e4 e8 e8 e8 e8 e4
+
+  % page 6, system 2
+  e4( d2.)
+  fis8 fis8 fis8 g8 fis4 e4
+  g4 fis4 e4 c4
+  r2 r4 e8 r8
+
+  % page 7, system 1
+  r2 e4( g4)
+  fis2
+  e8 e8~ e8 c8
+  \time 3/2
+  c1~ c4 r4\fermata
+
+  % page 7, system 2
+  \time 2/2 \tempo 2=60
+  fis'2 fis4 e4
+  gis4 b4 fis16 e8.~ e4
+  gis4 fis4 e4( cis8) cis8
+
+  % page 8, system 1
+  e8 e8 cis4 cis16 b8.~ b4
+  \slashedGrace e16( fis2) fis4 e4
+  gis4 b4 fis16 e8.~ e4(
+
+  % page 8, system 2
+  gis4) fis8 e8 e8 e8 cis8( e8)
+  e1~
+  e1\fermata
+}
+
+altoOneMusic = \relative c' {
+  \key c \major
+  \numericTimeSignature
+  \time 2/2
+  \tempo 2=55
+  \set Score.markFormatter = #format-mark-circle-numbers
+
+  % lead-in
+  b1
+  r1
+
+  % page 3, system 1 (solo)
+  r4 b4 b4 cis
+  e4 e8 e8 e8 e8 e4
+  e1\fermata
+  fis8 fis8 fis8 gis fis4 e4
+  
+  % page 3, system 2 (solo)
+  gis4 fis4 e4\fermata cis4
+  b4 gis4 b4~ b8 cis8
+  e4 cis4 e4 gis4
+  fis2 e8 e8~ e8 cis8
+  b1
+  
+  % page 3, system 3 (duet)
+  \slashedGrace e16( fis2) fis4 e4
+  gis4 b4 fis8 e8~ e4
+  gis4 fis4 e4( cis8) dis8
+  e8 e8 cis4 b2
+  
+  % page 3, system 4 (duet)
+  \slashedGrace e16( fis2) fis4 e4 gis4 b4 fis8 e8~ e4
+  gis4 fis8 e8 e8 e8 cis4
+  e1~
+  e1
+
+  % page 4, system 1
+  r4 b4 b4 cis4
+  e4 gis8 a8 gis8 fis8 e4
+  fis4( e4 dis4 cis4)
+  dis8 e8 fis8 gis8 fis4 gis4
+
+  % page 4, system 2
+  a4 a4 gis4 gis4
+  gis4 e4 gis4~ gis8 a8
+  gis4 e4 gis8( a8) b8( cis8)
+  b2 a4 a8 a8
+
+  % page 5, system 1
+  gis1
+  gis2 gis4a4
+  b4 b4 a8( gis8~ gis4)
+  gis4 a4 b4 a8 a8
+
+  % page 5, system 2
+  gis8 a8 b4 a4( gis4)
+  \slashedGrace e16( fis2) fis4 e4
+  gis4 b4 fis16( e8.~ e4)
+  gis4 fis8 e8 e8 e8 cis4
+
+  % page 6, system 1
+  b2. r4
+  \time 4/4 \tempo 4=90
+  r1
+  r4 b4 b4 c4
+  e4 d8 d8 c8 c8 c4
+  
+  % page 6, system 2
+  b4( c4 b4 c4)
+  c8 d8 e8 e8 d4 c4
+  e4 e4 e4~ e8 c8
+  b4 g4 b4~ b8 c8
+
+  % page 7, system 1
+  e4 c4 e2(
+  fis4) e4 d4 c4
+  \time 3/2
+  b1~ b4 r4\fermata
+
+  % page 7, system 2
+  \time 2/2 \tempo 2=60
+  r4 b'4 b4 b4
+  b4 a4 a4( gis4)
+  gis4 fis4 gis4~ gis8 gis8
```

Comment:
Almost missed this.  The dashed slurs/ties - I don't think they're actually slurs or ties.  I think they mean something like don't take a breath or something.  In this case (Alto voice, Page 7, system 2, last measure) - look at the words.  You can't tie the g# quarter note with the g# eighth note - you need to articulate two different words with those notes.  If we tie them, the rhythm in the MIDI won't match what's needed.  I think for creating MIDI's we should omit all dashed slurs.  Most of the time it's a slur and the rhythms will come out in the MIDI.  But in several places it changes the rhythm.  I won't identify more of those - scan through the music to find them and remove them.